CSV/neoHeaders/ReviewerHeaders.csv

```diff
@@ -0,0 +1 @@
+reviewerId:ID(Reviewer),color:string,vip:boolean,firstName:string,lastName:string
```

Comment:
Just going to comment these in general, very cool to see the formatting of Neo4j.